packages/rest-hooks/src/manager/DevtoolsManager.ts

```diff
@@ -25,20 +27,27 @@ export default class DevToolsManager implements Manager {
     config: DevToolsConfig = {
       name: `Rest Hooks: ${globalThis.document?.title}`,
       serialize: {
-        replacer: (key: string, value: unknown) => {
+        // the default options are only used if serialize isn't specified, so we include the default here
+        options: {
+          circular: '[CIRCULAR]',
+          date: true,
+        },
+        /* istanbul ignore next */
+        replacer: (key: string | number | symbol, value: unknown) => {
           if (
+            HASINTL &&
             typeof value === 'number' &&
```

Comment:
```suggestion
            (typeof value === 'number' || typeof value === 'bigint') &&
```

Reply:
I won't ever be using bigint for a timestamp. This is for human readable Date.now()

Reply:
a user wouldn't ever provide one? also the upcoming `Temporal.Now.instant().epochNanoseconds` is a bigint :-)

Reply:
These are not user fields - this is for my meta data